xonsh/built_ins.py

```diff
@@ -613,6 +613,10 @@ def run_subproc(cmds, captured=True):
             'obj': prev_proc,
             'bg': background
         })
+    try:
+        builtins.__xonsh_shell__.settitle()
+    except AttributeError:
+        pass
```

Comment:
Without the conditional (or the try/except), the `import` tests failed, so I stuck this in.

The problem is actually `builtins` not having `__xonsh_shell__` in that case, not `__xonsh_shell__` not having `settitle`.

I also tried setting a default (`None`) for `__xonsh_shell__` in `builtins.py`, but that didn't work, either (same error).  So maybe imports don't even initialize the builtins?  Not sure what was happening.  Or maybe it's more a quirk of `nosetests3` than anything else?